### DISCUSSION TITLE
Pikka 2.3.0

### DISCUSSION THREAD

**Discussion by rsayfutdinov** [rsayfutdinov/Pikka#8] [Announcements]
### Pikka 2.3.0 is live on AppStore

Version 2.3.0 brings a completely new magnifier view with the possibility to change the magnification level.

#### Picker magnification level
Change picker magnification level using +/- keys or from the app preferences.

[You can download it here](https://apps.apple.com/us/app/pikka-color-picker/id1195076754)

**Comment by floq-design** [comment]
Hi,
Just tried to (re)install previous purchase via the Apple Store and it's saying I need macOS v11?
Is that a bug or are you not supporting Catalina & older macOS anymore?
Cheers
James

**Reply by rsayfutdinov** [reply]
Hi James,

It is unfortunately not possible for me to support the older versions of macOS and provide a good app quality. So I've decided to drop support of old macOS versions. 

On what macOS version are you?

Best regards,
Ruslan.

**Reply by floq-design** [reply]
That is a shame. I'm on Catalina v10.15.7.

**Reply by rsayfutdinov** [reply]
> That is a shame. I'm on Catalina v10.15.7.

I can send you a custom App build for 10.5.7. But i cannot guarantee that everything will work as good as under macOS 11.

**Reply by floq-design** [reply]
That would be great - give it a go!
Thanks

**Reply by rsayfutdinov** [reply]
> That would be great - give it a go!
> 
> Thanks

Please reach me at support@1sense.app 
I give you a download link.

Regards,
Ruslan
